app/services/true_layer/importers/import_provider_service.rb

```diff
@@ -36,8 +34,6 @@ def mapped_resource
         {
           true_layer_response: provider,
           credentials_id: provider[:credentials_id],
-          token: applicant.true_layer_secure_data_id,
-          token_expires_at:,
```

Comment:
I couldn't find any instance of us doing anything with this data, besides asserting against it in tests.

Perhaps we intended to use it when retrieving bank data, or perhaps we wanted to reconcile failed imports that had expired tokens?

I wasn't able to glean anything from my digging, but maybe someone has some missing context about _why_ this data was stored on the bank provider record as well as the applicant.

Reply:
No missing context from me 🤷‍♂️